features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json

```diff
@@ -84,7 +84,7 @@
   "Apis.Details.InfoBar.more": "MORE",
   "Apis.Details.InfoBar.owner": "Owner",
   "Apis.Details.InfoBar.provider": "Provider",
-  "Apis.Details.Operations.notFound": "operations Not found",
+  "Apis.Details.Operations.notFound": "Operations Not found",
```

Comment:
"F" needs to be a block letter in the word "found"

Reply:
Fixed